sys/include/usb/usbus.h

```diff
@@ -232,7 +254,7 @@ typedef struct usbus_endpoint {
     struct usbus_endpoint *next;    /**< Next endpoint in the
                                          @ref usbus_interface_t list of
                                          endpoints */
-    usbus_hdr_gen_t *hdr_gen;       /**< Optional additional header generator */
+    usbus_descr_gen_t *descr_gen;     /**< Optional additional descriptor generator */
```

Comment:
Indentation level is broken here and in the equivalent lines below.

Reply:
All fixed, thanks